### PR TITLE
Ca 964 on sip call, reporter browser session isn't being muted

### PR DESCRIPTION
**Pull request by Galzk**

### CODE REVIEW

File: src/hooks/useIsSilenced/useIsSilenced.tsx [hunk 1]
```diff
@@ -2,41 +2,44 @@ import { useEffect } from 'react';
...
   } = useVideoContext();
   const { isSilenced, setIsSilenced, setNotification } = useAppState();
+  const [isAudioEnabled, toggleAudioEnabled] = useLocalAudioToggle();
+
   const participants = useParticipants();
-  const participantCommands = useParticipant();
 
   useEffect(() => {
     const isReporter = ParticipantIdentity.Parse(localParticipant.identity)['partyType'] === PARTICIPANT_TYPES.REPORTER;
+    if (!isReporter) return;
```
Comment (talr) [inline src/hooks/useIsSilenced/useIsSilenced.tsx:20]: I would add Logger.Info and Logger.End here.

File: src/hooks/useIsSilenced/useIsSilenced.tsx [hunk 2]
```diff
@@ -2,41 +2,44 @@ import { useEffect } from 'react';
...
       });
       console.log('You are silenced because of a zoiper call');
       setIsSilenced(true);
-      participantCommands.muteParticipant(localParticipant);
+      if (isAudioEnabled) toggleAudioEnabled();
     }
-    if (isSilenced && isReporter && !isSipClientConnected) {
+    if (isSilenced && !isSipClientConnected) {
       setNotification({ message: 'Zoiper call disconnected. Please unmute yourself' });
       console.log('Zoiper call disconnected, you are un-silenced');
```
Comment (talr) [inline src/hooks/useIsSilenced/useIsSilenced.tsx:39]: Can we user eScribers logger?

**Review by talr — CHANGES_REQUESTED** [review]

File: src/hooks/useIsSilenced/useIsSilenced.tsx (hunk 1 above)
Comment (Galzk) [reply src/hooks/useIsSilenced/useIsSilenced.tsx:20]: This is a hook, no need for logger

File: src/hooks/useIsSilenced/useIsSilenced.tsx (hunk 2 above)
Comment (Galzk) [reply src/hooks/useIsSilenced/useIsSilenced.tsx:39]: We don't have an active logger on the React app currently

**Pull request merged**